src/QuadForm/Herm/Genus.jl

```diff
@@ -1694,14 +1694,14 @@ function hermitian_local_genera(E, p, rank::Int, det_val::Int, min_scale::Int, m
       dets = Vector{Int}[]
       for b in g
         if mod(b[1], 2) == 0
-          push!(dets, [1, -1])
+          push!(dets, Int[1, -1])
         end
         if mod(b[1], 2) == 1
-          push!(dets, [hyperbolic_det^(div(b[2], 2))])
+          push!(dets, Int[hyperbolic_det^(div(b[2], 2))])
         end
       end
 
-      for d in cartesian_product_iterator(dets, inplace = true)# Iterators.product(dets...)
```

Comment:
Hmm, why?

Reply:
`inplace = true` is the default for the function so I do not see why keeping it

Reply:
Sorry, I forgot about that, you are right.